src/Tickets/Commerce/Order_Modifiers/API/Fees.php

```diff
@@ -377,60 +419,4 @@ protected function update_fees_for_ticket( $ticket_id, $fees ) {
 		$this->manager->delete_relationships_by_post( $ticket_id );
 		$this->manager->sync_modifier_relationships( $fee_ids, [ $ticket_id ] );
 	}
-
-	/**
-	 * Get the callback for adding fees to the ticket data.
-	 *
-	 * @since TBD
-	 *
-	 * @return callable The callback for adding fees to the ticket data.
-	 */
-	protected function get_single_ticket_data_callback(): callable {
```

Comment:
We don't need to remove these, they work well as-is and allow us to wrap the method calls as we need them.

Reply:
responded above.